packages/experiments/src/components/Text/Text.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { IStyle, IPalette, ISemanticColors } from '../../Styling';
+import { IThemedProps } from '../../Foundation';
+import { ITextProps, ITextStyles } from './Text.types';
+
+export const TextStyles = (props: IThemedProps<ITextProps>): ITextStyles => {
+  const { className, inline, theme, wrap, variant, family, weight, size, color, hoverColor } = props;
+  const { palette, semanticColors, typography } = theme;
+
+  return {
+    root: [
+      typography.variants[variant!] as IStyle,
+      {
+        display: inline ? 'inline' : 'block'
+      },
+      family && {
+        // TODO: How are language specific font families configured?
+        fontFamily: typography.families[family]
```

Comment:
If this family doesn't exist in `typography.families`, what is used?

Reply:
Nevermind, with the defaults it shouldn't be an issue :)

Reply:
it would evaluate as undefined (and would render the default variant.)